js/main.js

```diff
@@ -146,3 +203,9 @@ app.on('activate', function () {
         windowMgr.showMainWindow();
     }
 });
+
+app.setAsDefaultProtocolClient('symphony');
+
+app.on('open-url', function (event, url) {
+    handleProtocolAction(url);
```

Comment:
yes, it is triggered for both when the app is running and not running. The plist thing is taken care of automatically by the `app.setAsDefaultProtocolClient('symphony');` call.